style.css

```diff
@@ -141,4 +147,180 @@
     padding-top: 15px;
     font-weight: 500;
   }
+
+  .section-p2 {
+    width: 100%;
+    height: 2788px;
+    padding-top: 114px;
+  }
+
+  .cards-container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(4, 1fr);
+    grid-row-gap: 88px;
+    width: 90%;
+    margin: auto;
+  }
+
+  .card {
+    border: 1px solid rgba(235, 235, 255, 1);
+    width: 100%;
+    height: 552px;
+    margin: auto;
+    border-radius: 16px;
+    background-color: #fff;
+  }
```

Comment:
- [ ] kindly consider displaying the content of each card using flexbox to meet the below requirement 
> For each project
> * The content of the project is aligned with the flexbox